source/channels/make-calls.rst

```diff
@@ -0,0 +1,71 @@
+Make a call (beta)
+==================
+
+|enterprise| |cloud| |self-hosted|
+
+.. |enterprise| image:: ../images/enterprise-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in the Mattermost Enterprise subscription plan.
+
+.. |cloud| image:: ../images/cloud-badge.png
+  :scale: 30
+  :target: https://mattermost.com/download
+  :alt: Available for Mattermost Cloud deployments.
+
+.. |self-hosted| image:: ../images/self-hosted-badge.png
+  :scale: 30
+  :target: https://mattermost.com/deploy
+  :alt: Available for Mattermost Self-Hosted deployments.
+  
+You can use Mattermost calls for voice calling and screen sharing functionality in channels, group messages, and direct messages. It’s available and on by default in Mattermost Cloud. For self-hosted deployments it's available as a plugin and needs to be enabled <link to config doc>.
+
+- Use calls on desktop and mobile apps in addition to web browsers.
+- Share screen (not available on mobile), raise hands, chat in a thread, and even switch between products in the Mattermost suite (ie. Channels, Boards, and Playbooks) while on a call.
+
+To start a call, select **Start call** in the channel header. Any active team member in the channel can join a call, whether it's a public or private channel. If someone from outside of the organization wants to join a call you'll need to provide them with a guest account and add them to the channel. Users who are archived or not registered won't be able to join a call.
+
+Each call has a unique URL. You can share a call's URL to use in a meeting request or share with other team mates. The link is unique to each channel and does not change between calls. To access the link, hover over the call dialog in the channel and open the **More** menu. Select **Copy link**. The call link is valid for long as the channel is not archived or deleted.
+
+Limitations
+-----------
+
+- In Mattermost Cloud, up to eight participants in each channel can join a call.
+- In Mattermost self-hosted, the recommended maximum number of participants per server is 200. This can be changed in the ``config.json`` file.
```

Comment:
```suggestion
- In Mattermost self-hosted deployments, the recommended maximum number of participants, across all calls in all channels on the server, is 200. This can be changed in the ``config.json`` file.
```